src/LogViewer/chunk-worker.js

```diff
@@ -139,8 +167,8 @@ const toHtml = (index, metadata) => {
       const className = getAnsiClasses(part);
       
       return className ?
-        `<span class="${className}">${part.text}</span>` :
-        `<span>${part.text}</span>`;
+        escapeHtml`<span class="${className}">${part.text}</span>` :
```

Comment:
I didn't investigate why this didn't break, but I think only the contents of the tag need to be escaped, since the `span` is part of the HTML that is being rendered to the logviewer. This should be:

```js
const html = escapeHtml`${part.text}`;
      
return className ?
  `<span class="${className}">${html}</span>` :
  `<span>${html}</span>`;
```